site/content/docs/5.3/examples/e-shop/index.html

```diff
@@ -0,0 +1,1567 @@
+            <ul class="dropdown-menu">
+              <li><button class="dropdown-item active"><span class="visually-hidden">Sort by</span>Newest</button></li>
+              <li><button class="dropdown-item"><span class="visually-hidden">Sort by</span>Oldest</button></li>
+              <li><button class="dropdown-item"><span class="visually-hidden">Sort by</span>Cheapest</button></li>
+              <li><button class="dropdown-item"><span class="visually-hidden">Sort by</span>Most expensive</button></li>
+            </ul>
+          </div>
+
+          <div class="d-flex">
+            <button class="btn btn-secondary d-lg-none flex-shrink-1" style="width: 146px;" data-bs-toggle="offcanvas" data-bs-target="#filterOffcanvas" aria-controls="filterOffcanvas">
+              <svg class="me-1" width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                <use xlink:href="#filter"/>
+              </svg>
+              <span class="visually-hidden">Toggle</span>Filter
+            </button>
+
+            <form><div class="btn-group d-none d-md-flex ms-2" role="group">
+              <input type="radio" class="btn-check" name="displayOffers" id="gridDisplay" autocomplete="off" checked>
+              <label class="btn btn-icon btn-no-outline" for="gridDisplay" title="Grid">
+                <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                  <use xlink:href="#grid"/>
+                </svg>
+                <span class="visually-hidden">Grid display</span>
+              </label>
+              <input type="radio" class="btn-check" name="displayOffers" id="listDisplay" autocomplete="off">
+              <label class="btn btn-icon btn-no-outline" for="listDisplay" title="List">
+                <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                  <use xlink:href="#list"/>
+                </svg>
+                <span class="visually-hidden">List display</span>
+              </label>
+            </div></form>
+          </div>
+        </div>
+
+        <p class="pt-md-3 h4 mb-3">
+          <span class="text-primary">16 matches</span> / <span class="d-none d-xl-inline">9</span><span class="d-xl-none">6</span> displayed
+        </p>
+
+        <ul class="row row-cols-1 row-cols-md-2 row-cols-xl-3 gy-3 mb-3 list-unstyled">
+          <li class="card border-0 pt-2 pt-md-4">
+            <h3 class="visually-hidden">xxxxx xxxxxx xx xxx xxx</h3>
+            <div class="d-flex position-relative mb-2 align-items-end justify-content-between border-bottom border-primary fw-bold">
+              <div class="d-flex">
+                <div class="p-2 bg-primary text-nowrap">
+                  <svg class="me-1" width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                    <use xlink:href="#tick"/>
+                  </svg>Flash sale
```

Comment:
Not sure if this version in the DSM is older or newer?

![screenshot-system design orange com-2023 04 24-09_46_03](https://user-images.githubusercontent.com/92363071/233932198-5ea203b5-5cb2-491d-bc28-a5adeb65463e.png)

Reply:
I would say that you can keep the actual version previously developped until we spoke about it during next specs meeting. BTW it would be great to ask them to send a warning when they change the design like that.

Reply:
Coming from last specs meeting (05/02/2023):

- The design in my first comment is newer but it's not the final version so there is no need to tackle it now.
- We need to discuss a process for this type of edge case: should the designer ask if any developper is working on this page before a design update? Should we find away to add a 'Notify me when the content of this page has been modified/updated?' Should the designer open a car for the design and a distinct one for the design update? Other? Please note that moving to Figma will help us thanks to its versionning system.